additionalPosts-biblio.php

```diff
@@ -1,8 +1,13 @@
 <?php
-    /*
-        Template Name: Additional Posts Bibliotech
```

Comment:
I'm assuming that where these `Template Name: ****` lines are being deleted, it is because the files in question don't need that metadata? Page templates do, I know - but I don't think these do?